src/test-run/commands/actions.js

```diff
@@ -343,3 +343,26 @@ export class SwitchToMainWindowCommand {
         this.type = TYPE.switchToMainWindow;
     }
 }
+
+export class SetNativeDialogHandlerCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type          = TYPE.setNativeDialogHandler;
+        this.dialogHandler = {};
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'dialogHandler' }
```

Comment:
Oh, just figured out that we don't have initializer in command, which means native dialog handler can't be set from raw API. We need to move handler validation/initialization logic from `t.setNativeDialogHandler` here, e.g. like it's done for `selector`